modules/ztp-image-based-upgrade-procedure-steps.adoc

```diff
@@ -0,0 +1,187 @@
+// Module included in the following assemblies:
+// * edge_computing/image-based-upgrade/ztp-image-based-upgrade.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ztp-image-based-upgrade-procedure-steps_{context}"]
+= Performing an image-based upgrade on managed clusters at scale in several steps
+
+For use cases when you need better control of when the upgrade interrupts your service, you can upgrade a set of your managed clusters by using the `ImageBasedGroupUpgrade` CR with adding actions after the previous step is complete.
+After evaluating the results of the previous steps, you can move to the next upgrade stage or troubleshoot any failed steps throughout the procedure.
```

Comment:
Join the paragraph with the previous paragraph as this is how it shows on the preview.

Reply:
What is the next upgrade stage. Best to state it explicitly.

Reply:
Same as above, it can be different depending on where you are in the flow